src/main/jbake/content/docs/components/alerts/index.adoc

```diff
@@ -29,27 +29,6 @@ Other Java 8 compatible runtime environments _should_ work as well.
 Hawkular source code is managed with link:https://maven.apache.org/[Apache Maven]. A minimal version 3.2.x is needed
 to build Hawkular Alerts project.
 
-=== Wildfly 9
-
-http://wildfly.org/downloads/[Download a Wildfly 9.0.1.Final] archive and expand it somewhere on your disk.
-
-=== Apache Cassandra
```

Comment:
I have removed on purpose to address users to build the standalone distribution via -Pstandalone profile.
Some external users tried to install first a Cassandra and we packaged the embedded distribution on the profile.
Cassandra info is moved below in the configuration part.
But I agree that I should indicate the minimal version required, I'm going to update that in the PR.